documentation/docs/tooling/docker_private_network.md

```diff
@@ -106,4 +109,157 @@ Here is an example config that can be added:
   "messageLayer": {
     "tangleWidth": 10
   }}
-```
\ No newline at end of file
+```
+
+## Running with `docker-compose` directly
+
+To get an instance up and running on your machine make sure you have [Docker Compose](https://docs.docker.com/compose/install/) installed.
+
+Then you just need to run this command:
+```bash
+docker-compose build
+MONGO_DB_ENABLED=true docker-compose --profile drng --profile grafana up -d
+```
+
+> Note: Docker will build the GoShimmer image which can take several minutes.
+
+### Base components
+
+These services that are created by default with `docker-compose up -d`.
+
+#### Configuration
+
+- MONGO_DB_ENABLED: Determines if the analysis tools should use a MongoDB instance for storing analysis data. Defaults to `false`. **Required if using [`--profile grafana`](#grafana)**.
+- SNAPSHOT_FILE: The full path to the message snapshot file. Defaults to `./goshimmer/assets/7R1itJx5hVuo9w9hjg5cwKFmek4HMSoBDgJZN8hKGxih.bin`
+- GOSHIMMER_TAG: (Optional) The [iotaledger/goshimmer](https://hub.docker.com/r/iotaledger/goshimmer) tag to use. Defaults to `develop`.
+- GOSHIMMER_CONFIG: The location of the GoShimmer config file. Defaults to `./config.docker.json`.
+
+#### Peer master
+
+A node that is used to expose ports via the host and to have a single attachment point for monitoring tools.
+
+##### Volumes
+
+Docker Compose creates a `shimmerdb` volume to maintain a tangle even after tearing down the containers. Run `docker-compose down -v` to clear the volume.
+
+##### Example
+
+You can set the environment variable configuration inline as seen in this example.
+
+```bash
+GOSHIMMER_TAG=develop docker-compose up -d
+```
+
```

Comment:
This bit appears 3x; either delete the other two sections or adjust the examples accordingly.

Reply:
I moved the sections so the example appears with the configuration options instead of with each container.